classes/class.ilOpencastPageComponentPluginGUI.php

```diff
@@ -525,20 +525,24 @@ protected function setStyleFromProps(ilTemplate $tpl, array $properties)
         $tpl->setVariable('RATIO', $ratio);
         $tpl->setVariable('MAX-WIDTH', $properties[self::PROP_WIDTH]);
         $tpl->setVariable('MAX-HEIGHT', $properties[self::PROP_HEIGHT]);
-        if ($properties[self::PROP_RESPONSIVE] != false) {
-            $tpl->setVariable('WIDTH', 'width:100%;');
+        if (array_key_exists(self::PROP_RESPONSIVE, $properties)) {
```

Comment:
I suggest to use a Null Coalescing Operator:

```
if ((bool)($properties[self::PROP_RESPONSIVE] ?? false) === true) {
    $tpl->setVariable('WIDTH', 'width:100%;');
}
```

as well as a more strict type handling, and last but not least turning the logic from != false to === true which is more readable.